docs/README-AWS.md

```diff
@@ -1,92 +1,132 @@
 
-For more information, read the AWS [documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ways-to-add-notification-config-to-bucket.html) about creating an SQS event notifications for S3 buckets.
+### CloudWatch Logs subscription filter
 
-#### Notes on SQS queues
-The SQS queues you want to use as trigger must have a visibility timeout of 910 seconds, 10 seconds more than the Elastic Serverless Forwarder Lambda timeout.
+The Lambda function can ingest logs contained in the message payload of CloudWatch Logs events and send them to Elastic. The CloudWatch Logs service serves as a trigger for the Lambda function. When a new record gets written to a Kinesis data stream, the Lambda function triggers.
 
-**Kinesis Data Stream input:**
+You can set up separate CloudWatch Log groups for each type of log. The `es_datastream_name` parameter in the config file is mandatory. If this value is set to an Elasticsearch data stream, the type of log must be correctly defined with configuration parameters. A single configuration file can have many input sections, pointing to different CloudWatch Logs groups that match specific log types.
 
-The Lambda function supports ingesting logs contained in the payload of a Kinesis data stream record and sends them to Elastic. The Kinesis data stream serves as a trigger for the Lambda function. When a new record gets written to a Kinesis data stream the Lambda function gets triggered. Users will set up separate Kinesis data streams for each type of logs, The config param for Elasticsearch output `es_datastream_name` is mandatory. If the value is set to an Elasticsearch datastream, the type of logs must be defined with proper value configuration param. A single configuration file can have many input sections, pointing to different Kinesis data streams that match specific log types.
+## Deploying Elastic Serverless Forwarder
 
-**CloudWatch Logs subscription filter input:**
-The Lambda function supports ingesting logs contained in the message payload of CloudWatch Logs events. The CloudWatch Logs serves as a trigger for the Lambda function. Users will set up separate Cloudwatch log groups for each type of logs, The config param for Elasticsearch output `es_datastream_name` is mandatory. If the value is set to an Elasticsearch datastream, the type of logs must be defined with proper value configuration param. A single configuration file can have many input sections, pointing to different CloudWatch Logs log groups that match specific log types.
+### Deployment overview
 
+At a high level the deployment consists of the following 2 steps:
 
-### Deployment:
-At a high level the deployment consists of the following steps:
+1. Add appropriate integrations in Kibana - open *Management** > **Integrations**, select the **AWS category**, and select and add each integration
```

Comment:
Formatting issue: 

<img width="996" alt="Screen Shot 2022-06-28 at 2 28 44 PM" src="https://user-images.githubusercontent.com/10479155/176268899-c711300b-95da-44e7-b376-9776da746e3a.png">


```suggestion
1. Add appropriate integrations in Kibana - open **Management** > **Integrations**, select the **AWS category**, and select and add each integration.
```

Reply:
Thanks, have rewritten this section